- set main module name for escript
- disable warnings_as_errors as this prevents rebar3 from building
  with erlang/29

Index: rebar.config
--- rebar.config.orig
+++ rebar.config
@@ -14,7 +14,7 @@
 {escript_name, rebar3}.
 {escript_wrappers_windows, ["cmd", "powershell"]}.
 {escript_comment, "%%Rebar3 3.27.0\n"}.
-{escript_emu_args, "%%! +sbtu +A1\n"}.
+{escript_emu_args, "%%! +sbtu +A1 -escript main rebar3\n"}.
 %% escript_incl_priv is for internal rebar-private use only.
 %% Do not use outside rebar. Config interface is not stable.
 {escript_incl_priv, [{relx, "templates/*"},
@@ -49,8 +49,7 @@
         {erl_opts, [no_debug_info]},
         {overrides, [
             {override, erlware_commons, [
-                {erl_opts, [no_debug_info,
-                            warnings_as_errors]},
+                {erl_opts, [no_debug_info]},
                 {deps, []}, {plugins, []}]},
             {add, ssl_verify_hostname, [{erl_opts, [no_debug_info]}]},
             {add, certifi, [{erl_opts, [no_debug_info]}]},
